src/plugins/blocksmith/components/imgur-post-block-modal/getImgurPostMediaId.test.js

```diff
@@ -0,0 +1,36 @@
+import test from 'tape';
```

Comment:
@jalopez13 can you add a test for non-album embed code

Reply:
@efjacobson 

Added test for non-album embed.